openpype/modules/deadline/plugins/publish/submit_max_deadline.py

```diff
@@ -71,13 +73,10 @@ def get_job_info(self):
 
         job_info.Pool = instance.data.get("primaryPool")
         job_info.SecondaryPool = instance.data.get("secondaryPool")
-        job_info.ChunkSize = instance.data.get("chunkSize", 1)
+        job_info.ChunkSize = instance.data.get("chunkSize", self.chunk_size)
```

Comment:
Since we're getting into attribute definitions now coupling this very tightly to how an instance is created/collected I feel like should we be getting pretty certain that `chunkSize` will be defined on the instance.

I wonder whether we should be getting into the habit of not building fallback value on fallback value. The Collect Render plug-in is currently always setting the chunk size. The logic here now falling back to `self.chunk_size` and maybe even doing anything with regards to that might just end up being more confusing to trace back where the value came from.

At the very best case - if the value itself would not need explicit validation or alike then the attribute definition could just be directly on this Submission plug-in and the value get used one to one without needing to trace back to the Creator, other Collectors, etc. The value would then only really be useful for this particular plug-in. Which I think would be the best case because then it also only exists in the Deadline module for which it is intended. For example look at some of the 'changes' [here](https://github.com/ynput/OpenPype/pull/4388/files#diff-148e36a4d12835986c861be64a4320853ba6f30a15b9cd2f91f49e00b301362e) to Submit Maya Deadline.

I've mentioned it before but I'd actually even like to argue that MOST of the attribute definitions (and its usage) should be directly in Abstract Submit Deadline so that not every host needs to 'reimplement' this logic which is basically doing the same thing.